### PR TITLE
move group: fixing up sidebar and rewording

### PR DESCRIPTION
**Pull request by mlautman**
[live](http://tutorials.picknik.ai/doc/move_group_interface/move_group_interface_tutorial.html)

### CODE REVIEW

File: doc/move_group_interface/move_group_interface_tutorial.rst [hunk 1]
```diff
@@ -5,36 +5,39 @@ In MoveIt!, the simplest user interface is through the :move_group_interface:`Mo
 
 .. image:: move_group_interface_tutorial_start_screen.png
 
-Watch the `YouTube video demo <https://youtu.be/4FSmZRQh37Q>`_
```
Comment (davetcoleman) [inline doc/move_group_interface/move_group_interface_tutorial.rst]: Keeping this makes it quicker for users to get excited about the tutorial - why remove it?

File: doc/move_group_interface/move_group_interface_tutorial.rst [hunk 2]
```diff
@@ -5,36 +5,39 @@ In MoveIt!, the simplest user interface is through the :move_group_interface:`Mo
...
 Prerequisites
-^^^^^^^^^^^^^
+-------------
 If you haven't already done so, make sure you've completed the steps in `Prerequisites
 <../prerequisites/prerequisites.html>`_.
 
-Start Rviz and MoveGroup node
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Make sure you have re-sourced the setup files::
+The entire code
```
Comment (davetcoleman) [inline doc/move_group_interface/move_group_interface_tutorial.rst]: i disagree with this... running should be first, then explanation. its so much harder to understand what code is doing if you haven't seen its output yet

Also section heading should be capitalized

File: doc/move_group_interface/move_group_interface_tutorial.rst [hunk 3]
```diff
@@ -5,36 +5,39 @@ In MoveIt!, the simplest user interface is through the :move_group_interface:`Mo
...
+---------------
+The entire code can be seen :codedir:`here in the MoveIt! Github project<move_group_interface/src/move_group_interface_tutorial.cpp>`.
+
+.. tutorial-formatter:: ./src/move_group_interface_tutorial.cpp
+
+The launch file
+---------------
+The entire launch file is :codedir:`here<move_group_interface/launch/move_group_interface_tutorial.launch>` on github. All the code in this tutorial can be run from the moveit_tutorials package that you have as part of your MoveIt! setup.
```
Comment (davetcoleman) [inline doc/move_group_interface/move_group_interface_tutorial.rst]: whole repo: github -> Github

no need for "entire" launch file... its just a launch file

package name should be bold or code

File: doc/move_group_interface/move_group_interface_tutorial.rst [hunk 4]
```diff
@@ -50,9 +53,3 @@ Watch the `YouTube video demo <https://youtu.be/_5siHkFQPBQ>`_ for expected outp
  9. The object is removed from the environment.
 
 .. |B| image:: ./move_group_interface_tutorial_robot_with_box.png
-
-Explaining the Demo
```
Comment (davetcoleman) [inline doc/move_group_interface/move_group_interface_tutorial.rst]: keep this last

**Comment by mlautman** [comment]
@davetcoleman changes made and pushed to tutorials.picknik.ai

**Pull request merged**